gym_multigrid/envs/ctf.py

```diff
@@ -825,6 +830,17 @@ def __init__(
             [len(self.actions_set) for _ in range(self.num_blue_agents)]
         )
         self.ac_dim = self.action_space.shape
+        self.store_act_permuts = store_action_permutations
+
+        if self.store_act_permuts:
+            if self.num_blue_agents==1:
+                self.act_permuts = np.arange(5)
+            elif self.num_blue_agents==2:
+                self.act_permuts = cartesian_product(np.arange(5), np.arange(5))
```

Comment:
Instead of 5 as a magic number, the code should use the length of the available actions